### PR TITLE
fix: fix build:deps command on Windows

### PR DESCRIPTION
**Pull request by waterplea**

### CODE REVIEW

File: sdk/base/package.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
   "types": "./index.d.ts",
   "sideEffects": true,
   "scripts": {
-    "peggy": "peggy --allowed-start-rules '*' --plugin ./node_modules/ts-pegjs/dist/tspegjs -o lib/exver/exver.ts lib/exver/exver.pegjs",
+    "peggy": "peggy --allowed-start-rules * --plugin ./node_modules/ts-pegjs/dist/tspegjs -o lib/exver/exver.ts lib/exver/exver.pegjs",
```
Comment (dr-bonez) [inline sdk/base/package.json:7]: the single quotes are required on unix

File: sdk/base/package.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
     "test": "jest -c ./jest.config.js --coverage",
-    "buildOutput": "npx prettier --write '**/*.ts'",
+    "buildOutput": "npx prettier --write **/*.ts",
```
Comment (dr-bonez) [inline sdk/base/package.json:9]: same here

File: sdk/package/package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "test": "jest -c ./jest.config.js --coverage",
-    "buildOutput": "ts-node ./lib/test/makeOutput.ts && npx prettier --write '**/*.ts'",
+    "buildOutput": "ts-node ./lib/test/makeOutput.ts && npx prettier --write **/*.ts",
```
Comment (dr-bonez) [inline sdk/package/package.json:18]: same here

**Review by dr-bonez — CHANGES_REQUESTED** [review]

**Comment by dr-bonez** [comment]
@waterplea can you explain why you needed to remove the quotes?

**Comment by waterplea** [comment]
> @waterplea can you explain why you needed to remove the quotes?

It was failing on windows, saying no files matched the pattern, I believe.

**Review by dr-bonez — APPROVED** [review]

**Pull request merged**